gr-audio/lib/alsa/alsa_sink.h

```diff
@@ -29,6 +29,7 @@
 
 #include <gnuradio/audio/sink.h>
 #include <alsa/asoundlib.h>
+#include <volk/volk_alloc.hh>
```

Comment:
Move this to the .cc; you don't need any of the things declared in that header in the public header for the ALSA sink, but you need it in the implementation :)

Reply:
Sounds good. Thanks!